### PR TITLE
Docs cleanup + misc.

### PR DESCRIPTION
**Pull request by TechGlue**
Added the following to the doc. 
-  Action badges  
-  Description of the project (taken from our slides).
-  Links, to each view respective documentation and link to our demo video. 
-  Reorganized the contents of the doc.
 
I also made some changes to the python script if you can review my code and let me know of any improvements I can make that'd be great. Same with the doc let me know if I should re-word something or make changes to it. 

Thanks

### CODE REVIEW

File: initDatabase.py [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
+
+# Ensure that the courses.csv file present
+# By default all the courses will be posted into the database for testing purposes
+# Change the session wanted number to limit the number of created sessions
 sessions_wanted = 56
 courseResponseIds = []
 postHeaders={
 }
 
-# Make sure sessions wanted is less than the number of classes in the csv which is 56
```
Comment (glotchimo) [inline initDatabase.py:11]: Why not get the length of the CSV at runtime and set the max that way?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
 # Tutor Lab Queue System
+[![Docker Image CI](https://github.com/glotchimo/tlqs/actions/workflows/docker.yml/badge.svg)](https://github.com/glotchimo/tlqs/actions/workflows/docker.yml) [![Go ](https://github.com/glotchimo/tlqs/actions/workflows/go.yml/badge.svg)](https://github.com/glotchimo/tlqs/actions/workflows/go.yml)
+
 This is the monorepo for TLQS, containing a React SPA and a Go back-end. It features a Docker setup for quick containerized deployment and a Makefile for a rapid build-test-edit cycle.
 
+### What is TLQS?
+TLQS (pronounced “TALKS”) is a web-based application that supports students by facilitating a robust and streamlined way to interact with the tutors at EWU.
```
[the remark this answers is not in the document]
Comment (TechGlue) [reply README.md:7]: I like it. It's very techy.

File: initDatabase.py (hunk 1 above)
Comment (TechGlue) [reply initDatabase.py:11]: Good point!

**Review by glotchimo — APPROVED** [review]
LGTM thanks man

**Pull request merged**